Project.toml

```diff
@@ -46,7 +46,7 @@ Requires = "0.5.2"
 SpecialFunctions = "0.7.2, 0.8"
 StatsFuns = "0.8.0"
 Tracker = "0.2.3"
-Zygote = "~0.3.4"
+Zygote = "~0.3.4, 0.4"
```

Comment:
```suggestion
Zygote = "0.4"
```

Reply:
So are we alright to just drop 0.3.4?

Reply:
I think so, yeah. But this PR is superseded by #955, so we should probably move the discussion over there :+1: